reconcile/utils/oc.py

```diff
@@ -890,7 +869,7 @@ def add_group_kind(kind, kgv, new, preferred):
             group = new.split('/', 1)[0]
             new_group = True
             for pos in range(len(kgv[kind])):
-                if group in kgv[kind][pos]:
```

Comment:
Why would we do this?  The group should be explicit.

Deployment != DeploymenConfig for example.